src/hscroll.cxx

```diff
@@ -72,7 +72,7 @@ hscroll::Paint() const
 
 	/* set stored attributes and color */
 	attr_t old_attrs;
-	short old_pair;
+	short old_pair = 0;
```

Comment:
What's this change?

Reply:
True, it doesn't belong here at all, and the warning is bogus.

Reply:
Ok. I'll drop this pull request then and close the ticket - and submit pull request with just this variable initialization.

The warning is bogus, but should be silenced anyway.